Peer/clientlogger.cpp

```diff
@@ -6,13 +6,14 @@ ClientLogger* ClientLogger::Instance() {
 }
 const char* ErrorValueNames[] = {GET_NAME(ERROR), GET_NAME(SUCCESS),
```

Comment:
Add DEBUG type. It will be also useful when you need some additional information.